src/components/label/label.e2e.ts

```diff
@@ -174,4 +175,39 @@ describe("calcite-label", () => {
 
     expect(eventDetail).toBeTruthy();
   });
+
+  describe("label click behavior", () => {
+    it("works when label is rendered before input", async () => {
+      const page = await newE2EPage();
+      await page.setContent(html` <calcite-label></calcite-label> `);
+      await page.$eval(
+        "calcite-label",
+        (node) => ((node as HTMLElement).innerHTML = `<calcite-switch></calcite-switch>`)
+      );
+      await page.waitForTimeout(100);
+      await page.click("calcite-label", { clickCount: 1 });
+    });
+
+    it("works when label is rendered after input", async () => {
+      const page = await newE2EPage();
+      await page.setContent(html` <calcite-switch id="switch"></calcite-switch> `);
+      await page.waitForTimeout(100);
+      await page.evaluate(() => `<calcite-label for='switch'></calcite-label>`);
```

Comment:
@Elijbet this isn't setting anything on the document. You would need to do something like...

```
await page.evaluate(() => {
const label = document.createElement("calcite-label");
label.setAttribute("for", "switch");
document.body.appendChild(label);
});
```

Reply:
Should it also test the clicking after page.evaluate is executed?

Reply:
Maybe after page.evaluate you can use page.find() to make sure the label is there and has the correct for attribute?